core/frontend/src/test/scala/io/udash/bindings/inputs/SelectTest.scala

```diff
@@ -0,0 +1,313 @@
+      val options = Seq("A", "B", "C", "D", "E")
+      val p = Property[String]("B")
+
+      val select = (Select(p, options, Select.defaultLabel)(): @silent).render
+
+      select.childElementCount should be(5)
+
+      for (i <- options.indices) {
+        val o = options(i)
+        select.value = o
+        select.onchange(null)
+        p.get should be(o)
+      }
+    }
+
+    "synchronise state with property changes" in {
+      val options = Seq("A", "B", "C", "D", "E")
+      val p = Property[String]("B")
+
+      val select = Select(p, options.toProperty)(Select.defaultLabel).render
+
+      select.childElementCount should be(5)
+      select.value should be("1")
+
+      for ((o, idx) <- options.zipWithIndex) {
+        p.set(o)
+        select.value should be(idx.toString)
+      }
+    }
+
+    "synchronise property with state changes" in {
+      val options = Seq("A", "B", "C", "D", "E")
+      val p = Property[String]("B")
+
+      val select = Select(p, options.toProperty)(Select.defaultLabel).render
+
+      select.childElementCount should be(5)
+
+      for ((o, idx) <- options.zipWithIndex) {
+        select.value = idx.toString
+        jQ(select).change()
+        p.get should be(o)
+      }
+    }
+
+    "synchronise visible options" in {
+      val options = SeqProperty("A", "B", "C", "D", "E")
+      val p = Property[String]("B")
+
+      val select = Select(p, options)(Select.defaultLabel)
+      val selectElement = select.render
+
+      selectElement.childElementCount should be(5)
+      p.listenersCount() should be(5)
+
+      for ((o, idx) <- options.get.zipWithIndex) {
+        p.set(o)
+        selectElement.value should be(idx.toString)
+      }
+
+      options.remove(2, 3)
+      selectElement.childElementCount should be(2)
+      p.listenersCount() should be(2)
+      p.get should be("A")
+      selectElement.value should be("0")
+
+      options.prepend("1", "2")
+      selectElement.childElementCount should be(4)
+      p.listenersCount() should be(4)
+      p.get should be("A")
+      selectElement.value should be("2")
+
+      options.clear()
+      selectElement.childElementCount should be(0)
+      p.listenersCount() should be(0)
+      p.get should be("A")
+      selectElement.value should be("")
+
+      options.append("x", "y", "z")
+      selectElement.childElementCount should be(3)
+      p.listenersCount() should be(3)
+      p.get should be("x")
+      selectElement.value should be("0")
+
+      select.kill()
+      p.listenersCount() should be(0)
+    }
+  }
+
+  "Select with multiple on" should {
+    "synchronise state with property changes (deprecated)" in {
+      val options = Seq("A", "B", "C", "D", "E")
+      val p = SeqProperty[String]("B")
+
+      val select = (Select(p, options, Select.defaultLabel)(): @silent).render
+
+      select.childElementCount should be(5)
+      select.childNodes(0).asInstanceOf[org.scalajs.dom.html.Option].selected should be(false)
```

Comment:
Maybe some aliased import for `Option` instead of repeating FQCN bajillion times?